### ISSUE TITLE
How to exactly get the data in place

### ISSUE THREAD

**Issue by agaldran** [IML-DKFZ/fd-shifts#2]
Hello again,

So I am still trying to get your framework to work before trying to extend it. First thing after setting the env variables straight, I tried to run `fd_shifts launch --model=devries --dataset=cifar10` just to see what happened. 

I got a `` error because there is no such file or directory as ".../fd_shifts_datasets/corrupt_cifar10/CIFAR-C/labels.npy'". Which is expected, since I did not download any data myself.

I wanted to ask, could you please provide the links to where I can download this and the other datasets, and/or some basic instructions? In the readme it only says how they should be stored locally, but it never says where to get them from, and I would like to have the exact same datasets as in the paper of course.

I tried looking in the paper, but I only found that you are using tiny-imagenet from the ODIN repository, that's all.

Thanks!

Adrian

**Comment by agaldran** [comment]
As a follow-up, I managed to find in Zenodo and download the two cifar-corrupted datasets, 10 and 100. Then I got another error on not finiding tinyimagenet:

```
Couldn't find any class folder in [...]/fd_shifts_datasets/tinyimagenet/test
```

I tried to download this one from the [ODIN github page](https://github.com/ShiyuLiang/odin-pytorch). I am assuming I should download Tiny-ImageNet (resize) and not Tiny-ImageNet (crop), so I download a tar file that, when uncompressed, resulted in a single folder with 10,000 images. However, from the readme, I should have a tinyimagenet folder and a tinyimagenet_resize folder. Also, your library seems to expect some sort of train/test division, and class folders, I think. Is there something I am doing wrong, or misunderstanding here?

I cannot proceed forward until I remove this error, but I am assuming I will run into the same troubles when I get past tiny-imagenet and I am asked about downloading the wild-animals datasets, won't I? Any help is much appreciated.

**Comment by tbung** [comment]
Thank you for your interest in our project!

Indeed, you need `tinyimagenet_resize` for CNN experiments and `tinyimagenet` for ViT experiments. The Wilds and breeds datasets are only used in their respective experiments. I'll write up some instructions on setting up the datasets including download links and get back to you shortly.

**Comment by tbung** [comment]
I've written up where to get the datasets [here](https://github.com/IML-DKFZ/fd-shifts/blob/main/docs/datasets.md). Please let me know if that resolves your issue or if I can be of further assistance.

**Comment by agaldran** [comment]
Hello Till,

Thanks for that, I think I've got it. At least I managed to download and get in shape the smaller datasets, I will deal with ImageNet later if some ideas I want to try work out on these.

Cheers
